### ISSUE TITLE
ImportError: DLL load failed: module was not found

### ISSUE THREAD

**Issue by kirbyhi5** [Kataiser/tf2-rich-presence#128]
![image](https://user-images.githubusercontent.com/12120291/92290380-57926480-ef2d-11ea-92fa-4e54973c48b3.png)

**Comment by Kataiser** [comment]
Thanks for the report, it seems your install is broken, there's at least one necessary file missing (can't tell which one(s) from this error message). Try deleting and reinstalling the program. It's also possible that some antivirus software is getting false positives and removing files (see #126).

**Comment by MrJwhat** [comment]
I have allowed all the file as well and I am still getting this issue. Reinstalled it and everything.

**Comment by Kataiser** [comment]
Yeah I'm at a loss for this one. The only ways one of these files can be missing if they were removed by the user, OS, antivirus software, etc, or they were never correctly created by the extractor. I see some interaction with external files (`D:\obj\windows-release\37win32_Release\msi_python\zip_win32\__init__.py`) in the other person's error log, which shouldn't happen at all. May I see your crash log, as well as the file(s) in the `log` folder?

**Comment by Kataiser** [comment]
Also maybe try using the other installer, so download and extract the .zip if you used the self extractor or vice-versa.

**Comment by Kataiser** [comment]
One other thing, install the [x86 Visual C++ redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=48145). Version 1.14 should have removed the dependence on that but it may still be needed for some reason.
